### ISSUE TITLE
Makefile: Restore Section for "make-full" to build CX binary with opengl support

### ISSUE THREAD

**Issue by SkycoinSynth** [skycoin/cx#431]
In the Makefile restore Section for "make-full" to build CX binary with opengl support.

Someone deleted the section of make file, for building cx with opengl support.

**Comment by SkycoinSynth** [comment]
@jdknives cbrom cannot test cx-games because we are not compiling in opengl in to cx anymore. And the make file "make full" command has been removed.

**Comment by SkycoinSynth** [comment]
@jdknives @cbrom 

Cbrom found that old command in make file was 

```
go build $(GO_OPTS) -tags="base cxfx" -o ./bin/cx github.com/skycoin/cx/cmd/cx
```

**Comment by jdknives** [comment]
Fixed by #467
